deploy/terraform/modules/app_tier/vm-webdisp.tf

```
@@ -0,0 +1,112 @@
+# Create Web dispatcher NICs
+resource "azurerm_network_interface" "web" {
+  count                         = local.enable_deployment ? var.application.webdispatcher_count : 0
+  name                          = "${upper(var.application.sid)}_web${format("%02d", count.index)}-nic"
+  location                      = var.resource-group[0].location
+  resource_group_name           = var.resource-group[0].name
+  enable_accelerated_networking = local.web_nic_accelerated_networking
+
+  ip_configuration {
+    name                          = "ipconfig1"
```

```suggestion
    name                          = "IPConfig1"
```
Maybe make the name consistent with vm-app/vm-scs 's

https://github.com/Azure/sap-hana/blob/4f83a659355d5b7b7ed3aeb8b454202111f015c9/deploy/terraform/modules/app_tier/vm-app.tf#L10